src/lib/kvdb_cmp_lib/kvdb_cmp_lib.ml

```diff
@@ -0,0 +1,73 @@
+open Core
+open Command
+open Command.Let_syntax
+
+(* returns true if all the dbs are equal as K-V pairs *)
+let check_db_equalities db_dirs =
+  let dbs = List.map db_dirs ~f:Rocksdb.Database.create in
+  let alists = List.map ~f:Rocksdb.Database.to_alist dbs in
+  match alists with
+  | representative :: rest ->
+      List.for_all
+        ~f:([%equal: (Bigstring.t * Bigstring.t) list] representative)
+        rest
+  | [] ->
+      failwith "impossible"
+
+let test_db_equality ~modify_fn =
```

Comment:
Would be nice if you put this test helper into a inline test module so that it isn't compiled into the code and exposed. If we do want to share this code between files, it's best practice to put it into a `For_tests` submodule or something.